Calculator/Calculator.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+public class Calculator {
+    public static void main(String[] args) {
+        double firstNumber = 0.0;
+        double secondNumber = 0.0;
+        char operator = '+';
+        Double answer = null;
```

Comment:
Why you are using Double instead of double? As Double is class, double - primitive type

Reply:
I'm using Double because I need to assign null to variable

Reply:
Then don't do this) assign null is bad practice, as you don't need this